src/Desktop/UIAutomation/CustomObjects/CustomProperty.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using System;
+
+namespace Axe.Windows.Desktop.UIAutomation.CustomObjects
+{
+    public class CustomProperty
+    {
+        /// <summary>The RFC4122 globally unique identifier of this property.</summary>
+        [JsonProperty("guid")]
+        #pragma warning disable CA1720 // Identifier contains type name: name from JSON
+        public Guid? Guid { get; set; }
+        #pragma warning restore CA1720 // Identifier contains type name: name from JSON
+        /// <summary>A textual description of this property.</summary>
+        [JsonProperty("programmaticName")]
+        public string ProgrammaticName { get; set; }
+
+        /// <summary>The data type of this property's value as specified by the user, one of string, int, bool, double, point, or element.</summary>
+        // TODO Bill: add enum (with values member)
+        [JsonProperty("uiaType")]
+        public string UserType { get; set; }
```

Comment:
Again, let's don't overthink this. The name "Type" is sufficient to indicate the usage of this property within the context. And the property name within the class should match the json property name.

Reply:
The analyzer doesn't like `Type` as the name here

Reply:
Also, we'll want to distinguish between the type given by the user and the underlying UIA representation, but the latter isn't yet present in this PR.

Reply:
How is the type given by the user going to be different than the type from UIA?

Reply:
The user type is a string (from JSON). The type needed for registration is a `UIAutomationType`.

Reply:
How about `JsonType`?

Reply:
Discussed offline with @DaveTryon, changing to `DataType`.